### PR TITLE
docs: added jest documentation

### PR DESCRIPTION
**Pull request by serena97**
# Description

Added Jest Documentation. 

# Checklist:

- [x] My branch has been rebased onto the latest commit on main (don't merge main into your branch)
- [x] My commit messages are [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), for example: `feat(circleci): add support for nightly workflows`, `fix: set Heroku app name for staging apps too`

### CODE REVIEW

File: plugins/jest/readme.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+```yaml
+plugins:
+    - '@dotcom-tool-kit/jest'
+```
+
+## Options
+
+| Key | Description | Default value |
+|-|-|-|
+| `configPath` | Path to the [Jest config file](https://jestjs.io/docs/27.x/configuration) | N/A |
```
Comment (ivomurrell) [inline plugins/jest/readme.md:24]: **suggestion**: think we should clarify what the default behaviour is

```suggestion
| `configPath` | Path to the [Jest config file](https://jestjs.io/docs/27.x/configuration) | use Jest's own [config resolution](https://jestjs.io/docs/configuration/) |
```

File: plugins/jest/readme.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+## Tasks
+
+| Task | Description | Preconfigured hook |
+|-|-|-|
+| `JestLocal` | runs `jest` to execute tests | `test:local` |
+| `JestCI` | runs `jest` to execute tests in the CI with the `--ci` [option](https://jestjs.io/docs/cli#--ci) | `test:ci` |
+
+## To note
```
Comment (ivomurrell) [inline plugins/jest/readme.md:33]: **nitpick**: I think a heading like 'Tips' makes more sense but you can keep 'To note' if you prefer!
```suggestion
## Tips
```

**Review by ivomurrell — COMMENTED** [review]
This is great with just the right amount of information!

File: plugins/jest/readme.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+| Task | Description | Preconfigured hook |
+|-|-|-|
+| `JestLocal` | runs `jest` to execute tests | `test:local` |
+| `JestCI` | runs `jest` to execute tests in the CI with the `--ci` [option](https://jestjs.io/docs/cli#--ci) | `test:ci` |
+
+## Tips
+
+A common use case is to configure `test:local` and `test:ci` in your `.toolkitrc.yml` to run the `Eslint` task then the relevant Jest task: 
+```yaml
```
Comment (joelcarr) [inline plugins/jest/readme.md:36]: **nitpick (non-blocking):**  Add a new line above this code block to match spacing for the rest of the document.

**Review by joelcarr — APPROVED** [review]
Great documentation Serena! I have added a a small nit pick comment about some whitespace but it is really trivial.

**Review by ivomurrell — APPROVED** [review]

**Pull request merged**